File: application.client/src/DTOs/CreateBookingDto.ts
import { WorkspaceType } from "../models/WorkspaceType"

export interface CreateBookingDto
{
  coworkingId: number,
  name: string,
  email: string
  workspaceType: WorkspaceType,
  seats: number,
  startDate: string,
  endDate: string,
  startTime: string,
  endTime: string
}

File: Application.Server/Models/DTOs/CreateBookingDto.cs
﻿using Application.Server.Models.CoworkingDatabase;

namespace Application.Server.Models.DTOs
{
    public class CreateBookingDto
    {
        public int CoworkingId { get; set; }
        public string Name { get; set; } = null!;
        public string Email { get; set; } = null!;
        public WorkspaceType WorkspaceType { get; set; }
        public int Seats { get; set; }
        public DateOnly StartDate { get; set; }
        public DateOnly EndDate { get; set; }
        public TimeOnly StartTime { get; set; }
        public TimeOnly EndTime { get; set; }
    }
}
